pkg/genericcomp/fluentd.go

```diff
@@ -0,0 +1,189 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package genericcomp
+
+import (
+	"fmt"
+
+	"github.com/verrazzano/verrazzano-operator/pkg/constants"
+	"github.com/verrazzano/verrazzano-operator/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const fluentdConf = "fluentd.conf"
+
+// FluentdConfiguration Fluentd rules for reading/parsing generic component log files
+const FluentdConfiguration = `<label @FLUENT_LOG>
```

Comment:
Is it possible to externalize the FluentdConfiguration to a file instead of embedding this pseudo-XML or whatever it is here in the code?

Reply:
wls and helidon components have the same pattern of having the configuration in the code.   I am not sure how the operator would read from a file if it was externalized.  In any case, I think the suggested change is not in the scope of this task.